plugins/ida/ida_main.ml

```diff
@@ -136,13 +136,103 @@ let loader path =
             else code, Memmap.add data mem sec) in
   Project.Input.create arch path ~code ~data
 
+let int64_to_word arch = 
```

Comment:
A constant width (32) was used in #550: I thought that this was more appropriate.